pyproject.toml

```diff
@@ -104,12 +171,31 @@ max-locals = 20
 [tool.pylint.similarities]
 ignore-imports = "yes"
 
-# isort configuration
+################################################################################
+# Formatting Configuration                                                     #
+################################################################################
+
+# see https://black.readthedocs.io/en/stable/usage_and_configuration/index.html
+
+[tool.black]
+include = '\.pyi?$'
+# If `exclude` is not set, `black` excludes all files listed in `.gitignore`.
+# The following option extends this list of ignored files.
+# see https://black.readthedocs.io/en/stable/usage_and_configuration/file_collection_and_discovery.html#gitignore
+extend-exclude = '''
+# A regex preceded with ^/ will apply only to files and directories in the root
+# of the project.
+/(
+    \.git
+    | \.hg
+)/
+'''
+
+# see https://pycqa.github.io/isort/docs/configuration/config_files.html
+# see https://pycqa.github.io/isort/docs/configuration/options.html
+
 [tool.isort]
-multi_line_output = 3
-include_trailing_comma = true
-force_grid_wrap = 0
-use_parentheses = true
-line_length = 88
+# see https://pycqa.github.io/isort/docs/configuration/profiles.html#black
+profile = "black"
```

Comment:
This combines all deleted options, but it will be kept up-to-date with `black`s behavior